test_hdf5_cve.sh

```diff
@@ -211,14 +206,11 @@ cve-2024-0116-001.h5
 "
 
 GIF2H5_TEST_FILES="
-cve-2018-17433.h5
-cve-2018-17436.h5
-cve-2018-17438.h5
-cve-2018-17439.h5
-cve-2020-10809.h5
-cve-2022-25942.h5
-cve-2022-25972.h5
-cve-2022-26061.h5
```

Comment:
Looks like a few files were missed

Reply:
Oh, the three 2022 files don't exist.  They issues were from Talos and they didn't provide the POC files.
The other file was a gif file so it was renamed to cve-2020-10809.